kong/tools/grpc.lua

```diff
@@ -3,6 +3,7 @@ local protoc = require "protoc"
 local pb = require "pb"
 local pl_path = require "pl.path"
 local date = require "date"
+local loader = require("luarocks.loader")
```

Comment:
One of the things we're trying to do in our internal ticket is removing Luarocks from our builds, in order to increase security. I believe that the current way we want to do it is by protecting the `luarocks` executable only, so requiring luarocks parts like this should be fine. Just raising this as a point of possible failure.

Reply:
@kikito Yes, this should be fine without the executable. And thanks for sharing that info: FYI I am considering splitting `luarocks.loader` into a separare rock/library in LuaRocks 4.0 (meaning the loader only would become a dependency). The fact that the loader and executable have different security implications is further evidence for going ahead with this split!

Reply:
@kikito Yes, this should be fine without the executable. And thanks for sharing that info: FYI I am considering splitting `luarocks.loader` into a separare rock/library in LuaRocks 4.0 (meaning the loader only would become a dependency). The fact that the loader and executable have different security implications is further evidence for going ahead with this split!